HousingSearchApi/V1/Controllers/GetAssetListController.cs

```diff
@@ -74,22 +74,14 @@ private static bool RequestIncludesOtherSortParameters(GetAssetListRequest reque
         [LogCall(Microsoft.Extensions.Logging.LogLevel.Information)]
         public async Task<IActionResult> GetAllAssetList([FromQuery] GetAllAssetListRequest request)
         {
-            try
```

Comment:
How come no more try-catch?
Was it redundant all along?

Reply:
It was doing way more harm than good - any time there was a server error it was showing up as a 400, so I couldn't find any errors in cloudwatch